### PR TITLE
Log message instead of create label for GDScript

### PR DESCRIPTION
**Pull request by MrBrain295**
Fixes: #881

### CODE REVIEW

**Comment by Richienb** [comment]
It is actually called `GDScript`

**Comment by MrBrain295** [comment]
Done ☑️

File: g/GDScript.gd [hunk 1]
```diff
@@ -1,4 +1 @@
-extends Label
-
-func _ready():
-  self.text = "Hello World"
+print("Hello World")
```
Comment (Richienb) [inline g/GDScript.gd:1]: Shouldn't this be executed in some sort of init function?

https://docs.godotengine.org/en/latest/getting_started/step_by_step/scripting_first_script.html

File: readme.md [hunk 2]
```diff
@@ -342,7 +342,7 @@ Make sure to see [contributing.md](/contributing.md) for instructions on contrib
 * [GAML](g/GAML.gaml)
 * [Gammaplex](g/Gammaplex.gammaplex)
 * [GASOIL](g/GASOIL.gasoil)
-* [GDScript](g/GDScript.gd)
+* [GDSript](g/GDScript.gd)
```
Comment (Richienb) [inline readme.md:345]:
```suggestion
* [GDScript](g/GDScript.gd)
```

**Review by Richienb — CHANGES_REQUESTED** [review]

File: g/GDScript.gd (hunk 1 above)
Comment (Xrayez) [reply g/GDScript.gd:2]: Yes, I suggest this:

```suggestion
extends Node2D

func _ready():
    print("Hello World")

```

Even though `Node` is the core class, `Node2D` should be used for most use cases, and people use Godot for 2D games mostly.

**Review by Xrayez — CHANGES_REQUESTED** [review]

File: g/GDScript.gd [hunk 3]
```diff
@@ -1,4 +1,2 @@
-extends Label
-
-func _ready():
-  self.text = "Hello World"
+func _init():
+print("Hello World")
```
Comment (Richienb) [inline g/GDScript.gd:2]:
```suggestion
	print("Hello World")
```

**Review by Richienb — CHANGES_REQUESTED** [review]

**Review by Xrayez — COMMENTED** [review]
The current version is OK, but `_init` is not a function typically used in scripts, it's `_ready`.

**Comment by Richienb** [comment]
Fixed https://github.com/leachim6/hello-world/commit/508d4f483bf8dd6f4f8807a78afbba26c71e79fd

**Pull request merged**